src/vaults/interfaces/vault-harvests-model.interface.ts

```diff
@@ -25,7 +25,10 @@ export class VaultHarvestsModel implements VaultHarvestsExtended {
 
   @Title('amount')
   @Description('amount of harvested token')
-  @Example(15.34534)
+  @Example({
```

Comment:
looking back at this, we should format this to human readable balances as `number`

Reply:
i'll need to shake all call chain, but makes sense